src/View/Admin/connected.php

```diff
@@ -0,0 +1,23 @@
+<?php 
+    $url = (isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] === 'on' ? "https" : "http") . "://$_SERVER[HTTP_HOST]$_SERVER[REQUEST_URI]"; 
+    $url = add_query_arg( array(
+        'cc-connect' => 'connected',
+    ), $url );
```

Comment:
Nitpick only, not a blocker. I feel like PHPCS would be flagging a handful of things here.

Mostly the double quotes around `https` and `http` when those would be fine with single quotes.

I do believe we should be doing security measures on the two $_SERVER indexes we're constructing the URL with, instead of outright trusting the values.